### PR TITLE
Faq lifecycle rule delete objects

### PR DESCRIPTION
**Pull request by dsabelni**

### CODE REVIEW

File: umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.. _obs_faq_0199:
+
+Lifecyle Policy as Best Practice to Delete Objects and to Deal with Versioning
```
Comment (vladimirhasko) [inline umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst:3]: as this is FAQ should be stylized as question

File: umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Typical Use Cases
+-----------------
+
+A typical use case is recovering the latest historical version of an object by deleting the deletemarker version. This operation is recommended to be performed either through API or SDK. This is a single operation based on the specific object ID. For more information see `Versioning for Deleting an Object API request <https://docs.otc.t-systems.com/object-storage-service/api-ref/apis/operations_on_objects/deleting_an_object.html#versioning>`_.
+
+Another typical use case is deleting all versions of objects with a specific name (or even all objects from a bucket to delete the bucket). This operation however would need to be performed on multiple objects and thus can again run into difficulties based on the number of objects. It is recommended in this case to limit the batches of objects the operation runs on at the time and also the number of concurrency. For more information see `Deleting Objects <https://docs.otc.t-systems.com/object-storage-service/api-ref/apis/operations_on_objects/deleting_objects.html>`_.
+
+Configuring Lifecylce Policy for Mass Deleting Objects
```
[the remark this answers is not in the document]
Comment (dsabelni) [reply umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst]: Fixed

File: umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+------------------------------------------------------
+
+The best practice for mass deleting objects (regardless of versioning) is using lifecylce policy.
```
[the remark this answers is not in the document]
Comment (dsabelni) [reply umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst]: Fixed

File: umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst (hunk 1 above)
Comment (dsabelni) [reply umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst]: Fixed

**Review by vladimirhasko — APPROVED** [review]

**Review by marmulle2 — APPROVED** [review]

**Pull request merged**